### PR TITLE
Adding analyzer feedback for `tim-from-marketing` concept exercise

### PR DESCRIPTION
**Pull request by manumafe98**
# pull request

closes [#2687](https://github.com/exercism/java/issues/2687)

This could be an [example](https://exercism.org/mentoring/automation/cd618e6f14334c5ba7f87a6a2f3612c2/edit?source%5Bcriteria%5D=tim&source%5Bpage%5D=2&source%5Btrack_slug%5D=java) that triggers both comments

Reviewer Resources:

[Track Policies](https://github.com/exercism/java/blob/main/POLICIES.md#event-checklist)

### CODE REVIEW

**Comment by sanderploegsma** [comment]
The example you linked is an interesting case: clearly the author has enough Java knowledge to understand the Nullable concept, or they wouldn't have chosen to use `java.util.Optional`. In these cases I'm not sure they will appreciate if the analyzer is being a wise-ass by pointing out that they should use `null` instead. On the other hand, the goal of this exercise is to practice using `null`. 

I think I would prefer it if we leave an `actionable` comment that acknowledges that using `java.util.Optional` is a great way to deal with nullable arguments, but nudges the author to also try to solve the exercise without it. Once #2555 is completed, we could even point out that optionals are covered in a later concept.

What do you think?

**Comment by manumafe98** [comment]
> I think I would prefer it if we leave an `actionable` comment that acknowledges that using `java.util.Optional` is a great way to deal with nullable arguments, but nudges the author to also try to solve the exercise without it. Once #2555 is completed, we could even point out that optionals are covered in a later concept.
> 
> What do you think?

Sure, I like that!

File: exercises/concept/tim-from-marketing/.meta/design.md [hunk 1]
```diff
@@ -18,3 +18,17 @@
 
 - `strings`: strings will be compared to null and basic methods from strings will be called.
 - `if-else-statements`: using a conditional statement.
+
+## Analyzer
+
+This exercise could benefit from the following rules in the [analyzer]:
+
+- `essential`: If the student did not use the `null` expression in the `print` method, instruct the student to do so.
+- `actionable`: If the solution uses `Optionals` to solve the exercise, encourage the student to try solving it differently.
```
Comment (sanderploegsma) [inline exercises/concept/tim-from-marketing/.meta/design.md:27]:
```suggestion
- `actionable`: If the solution uses `Optionals` to solve the exercise, encourage the student to try solving it using `null` instead.
```

File: exercises/concept/tim-from-marketing/.meta/design.md [hunk 2]
```diff
@@ -18,3 +18,17 @@
 
 - `strings`: strings will be compared to null and basic methods from strings will be called.
 - `if-else-statements`: using a conditional statement.
+
+## Analyzer
+
+This exercise could benefit from the following rules in the [analyzer]:
+
+- `essential`: If the student did not use the `null` expression in the `print` method, instruct the student to do so.
```
Comment (sanderploegsma) [inline exercises/concept/tim-from-marketing/.meta/design.md:26]: Note that this comment currently would still apply to the example solution you linked to in the PR. 

To be honest I think we can remove this comment completely now that we have the one covering optionals: I don't think there is a way to pass the tests without using either `null` or `java.util.Optional`.

**Review by sanderploegsma — CHANGES_REQUESTED** [review]

File: exercises/concept/tim-from-marketing/.meta/design.md (hunk 2 above)
Comment (manumafe98) [reply exercises/concept/tim-from-marketing/.meta/design.md]: 👍

**Review by sanderploegsma — APPROVED** [review]
